### ISSUE TITLE
525 supplement note

### ISSUE THREAD

**Issue by CECSpecialistI** [uwlib-cams/MARC2RDA#202]
https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/5XX.csv

**Comment by cspayne** [comment]
Based on the [examples from loc](https://www.loc.gov/marc/bibliographic/bd525.html), this note indicates whether unnamed supplements/special issues are available for some issues or versions of this resource, which makes it seem like it is describing unnamed/uncataloged supplement works or expressions. RDA uses rdaw:P10172 "has supplement work" and rdae:P20172 "has supplement expression", so I think this note should be a note on work or expression - but is there a way to decide which to use or when one should be used over the other?

**Comment by CECSpecialistI** [comment]
I'm leaning towards mapping this as "has note on work" because it indicates that we have an augmentation aggregate, and we might not be minting entities for aggregating expressions to attach a note on expression to. What do you think, @GordonDunsire @AdamSchiff @lake44me ?

**Comment by tmqdeborah** [comment]
I agree with @CECSpecialistI that "has note on work" would be most appropriate for a 525 supplement note when the note indicates diachronicity, e.g., if the record is describing:
- a successive aggregating work (e.g., a serial), where issues and supplements accumulate
- an integrating work, where supplements provide changes in the subject material, e.g https://www.amazon.com/Bankruptcy-Insolvency-Accounting-Practice-Procedure-ebook/dp/B000SIS3ZS/ref=sr_1_1?crid=1RGWRKUKNJZUS&dib=eyJ2IjoiMSJ9.a40nMBuxecrPTy5tV-S6PQwabBirEcgFI7RsqOChCDVMu0svIqbWpHDsVHpvcBQN_h9aDpp9ZDc87G-0dH4UqkH9ORJEsF7spErPbqKvsK5tlTEIngNRsuPtYWlLS1Co32_6u6GBK13QkXKXh1vpZglAaaBfydX-HPDGzuS4u6jfg0-epx1Y7POvB3W_3f2_bhKyzHw0YMpMZ7HQyzBhAphot1wv2B39zZA7OB8Tnxg.mpbzIWemjQiGNdpgWCKl66vfufYwB3DDV5a-W8EXVgo&dib_tag=se&keywords=Bankruptcy+and+insolvency+accounting&qid=1718291017&sprefix=bankruptcy+and+insolvency+accounting%2Caps%2C131&sr=8-1&asin=B000SIS3ZS&revisionId=db494824&format=2&depth=1

Note that LC provides a 525 that begins with "Kept up to date" in many records that are codes 000/07 'm' (monographic). Under RDA, this code should be 'i' (integrating) because the embodied content is changed over time, even though the carrier might be a bound volume (as in the case of the example above). Given how often this happens, there is probably PCC guidance somewhere about when to code 'i' vs 'm'. Does anyone know about such guidance?

This raises a question about supplements that are not included automatically as part of a subscription, such as the example in the link above. If we have to sign up to get the updates, then does that mean that there are two manifestations: one with added material in separately published supplements (diachronic) and one without those supplements (static)?

And, if the note does not begin with "Kept up to date by" and is in a record that does not describe either a successive aggregating work or an integrating (aggregating or multi-part) work, then the work might be static. In which case, the question is whether the note indicates that the supplementary manifestation contains supplementary content (e.g., illustrations) or contains a continuation of the content of a multi-part work; e.g., is this CD supplementary (aggregated) or integral (a part of a multi-part)?
300    $a 245 p. : $b ill. ; $c 25 cm. + $e 1 CD-ROM (4 3/4 in.)
525    $a One CD-ROM in pocket attached to inside back cover.

I think that we can map the 525:
- to 'note on work' if:
  - the 000/07 = 's' or 'i' 
  - or if the 525 begins with "Kept up to date by"
- but to 'note on manifestation' if:
  - the 525 does not begin with "Kept up to date by"

Or we can just always map it to 'note on manifestation' as a default, to keep it simple.

**Comment by AdamSchiff** [comment]
The PCC guidance on mode of issuance is at http://original.rdatoolkit.org/lcpschp0_lcps0-63.html.

Regarding the example with the CD-ROM in pocket: often it just contains the same content of the print manifestation, but in PDF or other electronic form.  Other times it may contain supplemental material.

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________
